### DISCUSSION TITLE
after clicking run the project is not showing up

### DISCUSSION THREAD

**Discussion by EronitaB** [Budibase/budibase#2445] [Help] [answered]
Hi there,
I don't know what happened with my project!
When I click run the proper view its not showing up!
What to do?
Does anyone have any idea?

<img width="1680" alt="Screen Shot 2021-08-24 at 11 23 11" src="https://user-images.githubusercontent.com/77198959/130593980-f493119c-19cd-4144-a481-c425e993926e.png">

<img width="1679" alt="Screen Shot 2021-08-24 at 11 24 41" src="https://user-images.githubusercontent.com/77198959/130592289-1f4ff893-2710-4844-8a10-3d31dfe953b7.png">

**Comment by shogunpurple** [comment, marked as answer]
@EronitaB we are going to need a bit more information:

- Are there any error messages in the dev console when you open your app/builder
- Is this a brand new app?
- Which version of budibase are you on?
- It looks like your budibase installation is running behind a reverse proxy on `budi.thequantix.com` - if you hit the budibase URL directly, does your app work?

Thanks

**Reply by EronitaB** [reply]
Hey @shogunpurple , I just solved the problem, I had a file which caused all this and I don't even know why.
Thanks for trying to help me,
Eronita.
